cloudproxy-ui/src/main.js

```diff
@@ -0,0 +1,13 @@
+import '@babel/polyfill'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/claffin/cloudproxy/pullRequest?prid=7329682)
```suggestion
import "@babel/polyfill"
```